src/uniprotkb/components/protein-data-views/VariationView.tsx

```diff
@@ -90,60 +181,89 @@ const VariationView = ({
+          </th>
+          <th>Provenance</th>
         </tr>
       </thead>
       <tbody>
-        {Array.from(transformedData.variants)
-          .sort(sortByLocation)
-          .map((variantFeature) => {
-            let position = variantFeature.start;
-            if (variantFeature.start !== variantFeature.end) {
-              position += `-${variantFeature.end}`;
-            }
-
-            return (
-              <Fragment key={variantFeature.protvistaFeatureId}>
-                <tr
-                  data-id={variantFeature.protvistaFeatureId}
-                  data-start={variantFeature.start}
-                  data-end={variantFeature.end}
-                >
-                  <td>{position}</td>
-                  <td>
-                    {variantFeature.wildType}
-                    {'>'}
-                    {variantFeature.alternativeSequence}
-                  </td>
-                  <td>
-                    {variantFeature.descriptions?.map((description) => (
-                      <div key={description.value}>
-                        {`${description.value} (${description.sources.join(
-                          ', '
-                        )})`}
-                      </div>
-                    ))}
-                  </td>
-                  <td>
-                    {variantFeature.association &&
-                    variantFeature.association.length > 0
-                      ? 'Y'
-                      : 'N'}
-                  </td>
-                </tr>
-                <tr
-                  data-group-for={variantFeature.protvistaFeatureId}
-                  data-start={variantFeature.start}
-                  data-end={variantFeature.end}
-                >
-                  <td>
-                    <div>
-                      <strong>Consequence: </strong>
-                      {variantFeature.consequenceType}
+        {filteredVariants.map((variantFeature) => {
+          let position = variantFeature.start;
+          if (variantFeature.start !== variantFeature.end) {
+            position += `-${variantFeature.end}`;
+          }
+
+          return (
+            <Fragment key={variantFeature.protvistaFeatureId}>
+              <tr
+                data-id={variantFeature.protvistaFeatureId}
+                data-start={variantFeature.start}
+                data-end={variantFeature.end}
+              >
+                <td>
+                  {Array.from(
+                    new Set(variantFeature.xrefs.map((xref) => xref.id))
+                  ).map((id, i) => (
+                    <Fragment key={id}>
+                      {i !== 0 && <br />}
+                      {id}
+                    </Fragment>
+                  ))}
+                </td>
+                <td>{position}</td>
+                <td className={styles.change}>
+                  {variantFeature.wildType}
+                  {'>'}
+                  {variantFeature.alternativeSequence || <em>missing</em>}
```

Comment:
Does `missing` imply something biologically or is it just no data? I ask as it's not immediately clear if it means there's a deletion or no data.

Reply:
I don't remember with one it is, I think we had a look in a past website meeting, and the examples we say all happened to be marked as `del` in other fields. In any case, Maria asked to display `missing` in these cases.

Reply:
Looking at this they have `Consequence: inframe deletion` which makes me think we should use the lowercase `d` to indicate this but it is instead placed on the `*` line which also has things like `Consequence: stop gained`

<img width="563" alt="Screenshot 2022-04-19 at 14 43 05" src="https://user-images.githubusercontent.com/4357962/164017419-0847b722-8ee2-4420-819f-dbf1fbdde127.png">

Reply:
I'm not sure what to do to be honest, because the visualisation puts it to the "*" line and I'm not sure what the reason behind it was.
Do we have an example where a variant ends up on the "d" line? In which case we would be able to compare the data behind it

Reply:
Seems the `d` line isn't used. I've looked at a dozen entries with loads of variants but the `d` line is never occupied. Maybe this issue should be included in the variants refactor post-release.